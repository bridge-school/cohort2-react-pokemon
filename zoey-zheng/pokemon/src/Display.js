import React from 'react';

export const Display = ({ img, name }) => {
  return (
    <div>
      <img src={img} alt="pokemon" />
      <p> {name} </p>
    </div>
  );
};
