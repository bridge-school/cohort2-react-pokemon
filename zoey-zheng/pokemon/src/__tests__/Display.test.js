import React from 'react';
import { Display } from '../Display.js';
import renderer from 'react-test-renderer';

describe('Display component tests', () => {
  it('renders display component', () => {
    const display = renderer.create(<Display />);
    expect(display.toJSON()).toMatchSnapshot();
  });
});
