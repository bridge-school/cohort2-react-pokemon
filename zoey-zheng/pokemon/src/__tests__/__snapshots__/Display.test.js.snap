// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display component tests renders display component 1`] = `
<div>
  <img
    alt="pokemon"
    src={undefined}
  />
  <p>
     
     
  </p>
</div>
`;
